src/tasks/installers/common-tasks.ts

```diff
@@ -94,6 +97,44 @@ export function createEclipseCheCluster(flags: any, kube: KubeHelper): Listr.Lis
   }
 }
 
+export function updateEclipseCheCluster(flags: any, kube: KubeHelper, command: Command): Listr.ListrTask {
+  return {
+    title: `Update the Custom Resource of type ${CHE_CLUSTER_CRD} in the namespace ${flags.chenamespace}`,
+    enabled: () => !!flags[CHE_OPERATOR_CR_PATCH_YAML_KEY],
+    task: async (ctx: any, task: any) => {
+      const cheOperatorCrPatchYamlPath = flags[CHE_OPERATOR_CR_PATCH_YAML_KEY]
+      if (existsSync(cheOperatorCrPatchYamlPath)) {
+        const crPatch: any = yaml.safeLoad(readFileSync(cheOperatorCrPatchYamlPath).toString())
```

Comment:
```
let crPatch: any = {}
 if (existsSync(cheOperatorCrPatchYamlPath)) {
    crPatch: any = yaml.safeLoad(readFileSync(cheOperatorCrPatchYamlPath).toString())
}
```

Reply:
Ok, I handled this section to make it working even without cr patch argument